plugins/FlashSales/src/EasyAbp.EShop.Products.Plugins.FlashSales.HttpApiClient/EasyAbp.EShop.Products.Plugins.FlashSales.HttpApiClient.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This project should rename to `EasyAbp.EShop.Products.Plugins.FlashSales.HttpApi.Client`.